client/push/push_manager.ts

```diff
@@ -17,28 +17,51 @@
 declare var navigator: any;
 
 export default class PushManager {
-  private isRegistered_: boolean = false;
 
-  constructor() {
-    if (navigator.serviceWorker) {
-      navigator.serviceWorker.register('/push_service_worker.js')
-        .then((registration: ServiceWorkerRegistration) => {
-          this.isRegistered_ = true;
-        })
-        .catch((error: Error) => {
-          // FIXME(zino): What should we do?
-        });
-    }
-  }
+  constructor() {}
 
   async isRegistered(): Promise<boolean> {
     // Not implemented yet
     return false;
   }
 
-  async register(key: string): Promise<boolean> {
+  async register(key: string): Promise<any> {
```

Comment:
What kind of key argument is going to be used for here? It seems there is no usage in this function.
FYI, Push subscription including endpoint will return by browser as you know, And subscription.keys also need by server so, should send to server